### PR TITLE
Added DSM-USM configuration

### PR DESCRIPTION
**Pull request by guyarb**
### What does this PR do?

Adds USM-DSM documentation

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by urseberry** [comment]
Docs review on https://datadoghq.atlassian.net/browse/DOCS-5534

File: content/en/data_streams/zero_instrumentation.md [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+If the operating system image is Ubuntu or Debian, add the following after `environment`:
+
+```yaml
+"dockerSecurityOptions": [
+  "apparmor:unconfined"
+]
+```
+
+If you are using load balancers with your services, enable additional cloud integrations to allow Universal Service Monitoring to discover cloud-managed entities.
+* Install the [AWS Integration][2] for visibility in AWS Load Balancer. You must also enable ENI and EC2 metric collection.
```
Comment (cswatt) [inline content/en/data_streams/zero_instrumentation.md:369]:
```suggestion
If you are using load balancers with your services, enable additional cloud integrations to allow Universal Service Monitoring to discover cloud-managed entities. Install the [AWS Integration][2] for visibility in AWS Load Balancer. You must also enable ENI and EC2 metric collection.
```

**Review by cswatt — CHANGES_REQUESTED** [review]

File: content/en/data_streams/zero_instrumentation.md [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+
+Then, add the following tags to each load balancer:
+```conf
+ENV=<env>
+SERVICE=<service>
+```
+
+[1]: /containers/amazon_ecs/?tab=awscli#run-the-agent-as-a-daemon-service
+[2]: /integrations/amazon_web_services/
```
[the remark this answers is not in the document]
Comment (guyarb) [reply content/en/data_streams/zero_instrumentation.md:377]: USM should not be mentioned here, I didn't remove all instances of USM

File: content/en/data_streams/zero_instrumentation.md [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+If you configure the `system-probe` with environment variables, as is common with Docker and ECS installations, pass the following environment variable to `system-probe`:
+
+```
+DD_SYSTEM_PROBE_DATA_STREAMS_ENABLED=true
+```
+
+{{% /tab %}}
+{{% tab "ECS" %}}
+
+For ECS, enable USM and the system probe with the following JSON task definition. Deploy the task definition as a [daemon service][1].
```
[the remark this answers is not in the document]
Comment (guyarb) [reply content/en/data_streams/zero_instrumentation.md]: https://github.com/DataDog/documentation/pull/18213#discussion_r1220795167

File: content/en/data_streams/zero_instrumentation.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (guyarb) [reply content/en/data_streams/zero_instrumentation.md:371]: switched to DSM

**Review by cswatt — APPROVED** [review]

**Pull request merged**